public/app/features/dashboard-scene/serialization/transformToV2TypesUtils.ts

```diff
@@ -3,7 +3,9 @@ import {
   VariableRefresh as VariableRefreshV1,
   VariableSort as VariableSortV1,
   DashboardCursorSync as DashboardCursorSyncV1,
+  DataTopic,
 } from '@grafana/schema';
+import { DataTransformerConfig } from '@grafana/schema/dist/esm/raw/dashboard/x/dashboard_types.gen';
```

Comment:
```suggestion
import { DataTransformerConfig } from '@grafana/schema';
```
Is this not exported from the index?